### PR TITLE
[DOCFIX] Update Readme

### PR DESCRIPTION
**Pull request by apc999**
- Add a logo
- Add shields logos for releases, docker pulls, twitter and documentation
- Cleanup the readme text
- Deprecate JIRA and point to Github issues

Screenshot:
screencapture-github-alluxio-alluxio-blob-f8e0ee6446b8b2ffbb4d3d15738ebed989b23fae-readme-md-2018-12-27-15_36_06

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,57 @@
+<div align="center">
+  <img src="http://www.alluxio.org/static/media/alluxio-logo-horizontal-tight.a41c789d.svg"><br><br>
```
Comment (aaudiber) [inline README.md]: I suggest making the image much smaller or removing it

File: README.md [hunk 2]
```diff
@@ -61,3 +98,16 @@ work to the project under the project's open source license. Whether or not you
...
 authority to do so.
+For more detailed step-by-step tutorials, please read 
+[Contribute to Alluxio](https://www.alluxio.org/docs/1.8/en/contributor/Contributor-Getting-Started.html). For [new contributor tasks](https://github.com/Alluxio/new-contributor-tasks), please limit 2 tasks per new contributor.
+
+## Useful Links
+
+- [Alluxio Open Source Website](http://www.alluxio.org/) 
+- [Alluxio Inc.](http://www.alluxio.com/)
+- [Alluxio Latest Release Document](https://www.alluxio.org/download/releases/)
+- [Master Branch Document](https://www.alluxio.org/docs/master/en/)
```
Comment (aaudiber) [inline README.md]: I'd remove this since latest release is pretty much always better.

File: README.md [hunk 3]
```diff
@@ -61,3 +98,16 @@ work to the project under the project's open source license. Whether or not you
 explicitly, by submitting any copyrighted material via pull request, email, or other means you agree
 to license the material under the project's open source license and warrant that you have the legal
 authority to do so.
+For more detailed step-by-step tutorials, please read 
+[Contribute to Alluxio](https://www.alluxio.org/docs/1.8/en/contributor/Contributor-Getting-Started.html). For [new contributor tasks](https://github.com/Alluxio/new-contributor-tasks), please limit 2 tasks per new contributor.
+
+## Useful Links
+
+- [Alluxio Open Source Website](http://www.alluxio.org/) 
+- [Alluxio Inc.](http://www.alluxio.com/)
```
Comment (aaudiber) [inline README.md]: How about "Project Website"/"Alluxio Inc. Website"?

File: README.md [hunk 4]
```diff
@@ -61,3 +98,16 @@ work to the project under the project's open source license. Whether or not you
...
+- [Alluxio Latest Release Document](https://www.alluxio.org/download/releases/)
```
Comment (aaudiber) [inline README.md]: Perhaps shorten to just "Documentation"

File: README.md [hunk 5]
```diff
@@ -61,3 +98,16 @@ work to the project under the project's open source license. Whether or not you
...
+- [Master Branch Document](https://www.alluxio.org/docs/master/en/)
+- [Releases](http://alluxio.org/releases/)
+- [Downloads](http://www.alluxio.org/download)
+- [User Mailing List](https://groups.google.com/forum/?fromgroups#!forum/alluxio-users)
```
Comment (aaudiber) [inline README.md]: with `alluxio-dev` deprecated, this is just "Mailing List" now

**Comment by apc999** [comment]
@calvinjia  do you think building Alluxio with Gradle/Ant/SBT is relevant here? are they actually tested? I kinda incline to remove them but leaving only Maven. What's your thoughts?

File: README.md (hunk 2 above)
Comment (apc999) [reply README.md]: done

**Comment by apc999** [comment]
@aaudiber PTAL

File: README.md [hunk 6]
```diff
@@ -61,3 +99,14 @@ work to the project under the project's open source license. Whether or not you
 explicitly, by submitting any copyrighted material via pull request, email, or other means you agree
 to license the material under the project's open source license and warrant that you have the legal
 authority to do so.
+For a more detailed step-by-step guide, please read
+[how to contribute to Alluxio](https://www.alluxio.org/docs/1.8/en/contributor/Contributor-Getting-Started.html).
+For new contributor, please take 2 [new contributor tasks](https://github.com/Alluxio/new-contributor-tasks).
```
Comment (aaudiber) [inline README.md]: now that we're moving to issues and deprecating JIRA, should we remove this section?

**Pull request merged**